.github/workflows/release-windows.yml

```diff
@@ -0,0 +1,41 @@
+name: release-windows
+
+on:
+  release:
+    tags:
+      - '*'
+
+jobs:
+  windows:
+    name: windows-build
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: CMake build
+        run: |
+             cmake .
+      - name: Setup MSBuild.exe
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: MSBuild and tests
+        run: |
+             msbuild INSTALL.vcxproj /p:Configuration=Release
+      - name: Create binary archive from build
+        run: |
+             mkdir libuv-${{ github.event.release.tag_name }}\bin
+             mkdir libuv-${{ github.event.release.tag_name }}\lib
+             move include libuv-${{ github.event.release.tag_name }}\
+             move Release/uv_a.lib libuv-${{ github.event.release.tag_name }}\lib\
```

Comment:
this is renamed now. `cmake install` might be more reliable for this?

Reply:
I thought `cmake` usually can be used as a driver for the underlying build program (and vice versa). I forget the exact invocation, but we use it already in several of the CI scripts. (indeed, we even now have a short-term upload of windows artifacts installed by cmake, as seen here https://github.com/libuv/libuv/actions/runs/3961926739)

But the lack of ongoing need is basically why we don't have this. It has been requested and proposed similar to this several times, but there has not seemed to be an active an on-going involvement by someone who can join the @libuv/windows team to help me maintain it.

Reply:
> as seen here https://github.com/libuv/libuv/actions/runs/3961926739

I have no formulary with cmake neither, I just used what the repo readme instructed. But it seems that instruction now is for installing on system, not for producing a zip file ready for next stage for uploading. Maybe for making **artifacts**. 

And that's the problem of making **artifacts** there is a stated expiration time to deal with, you need a team for that. 

Making a **release** has a different GitHub Action process, and separate location that's more part of repo itself. You would have to manually delete it. And you can easily programmability access the file, which should be zipped. 

I can help out on maintaining a Windows binary availability. But I see a different process, whoever decides a release is ready, just create a tag, doing so produces a **zipped** binary available for Windows, shows up under the **release page** along with the other **two** archive files.

I find that all Linux distributions include some version of Libuv. The project I'm working on, I build the current `Libuv Linux binary` and added directly to repo. Found out when using with **PHP FFI** it needed to be in users home directory or regular platform's OS directory. Windows had no restrictions. 

I decide to just use whatever Linux system has, deleted the files and GitHub Action to produce them, https://github.com/symplely/uv-ffi/commit/f9ca562b5174dca4b089fa1411fb8d2f33a730e8